docs/source/key_components/geographic_visualization.rst

```diff
@@ -0,0 +1,201 @@
+Geographic Visualization
+=======================
+
+Env-geographic tool aims to provide users with an intuitive understanding
+of Maro's experimental data and help users better understand Maro's scenarios and working processes.
+Currently, Env-geographic supports two modes, which are real-time mode and local mode.
+The former mode shows users real-time experimental data and helps users judge the effectiveness of the model.
+The latter mode allows users to freely view the data after the experiment,
+helping users to view the effect of the experiment and make subsequent decisions.
+
+
+Dependency
+----------
+
+Env-geographic's startup depends on docker. 
+Therefore, users need to install docker on the machine and ensure that it can run normally.
+User could get docker through `Docker installation <https://docs.docker.com/get-docker/>`_.
+
+
+How to Use?
+-----------
+
+Env-geographic has 3 parts: front-end, back-end and experiment database. To start this tool,
```

Comment:
Users need 2 steps to start this tool:
1. start the database and choose the experiment name;
2. start the service with `maro inspector geo` in command line.

Then, organize the content like below?

Start database
\~~~~~~~~~~

Choose an existing experiment
^^^^^^^^^^^^^^^^^^^^^^

  Create a new experiment
^^^^^^^^^^^^^^^^^^

Start service
\~~~~~~~~~~